src/features/stats/reducer.js

```diff
@@ -82,11 +83,11 @@ const statsReducer = (state = initialState, { type, payload }) => {
             const previous_dex = calculateModifier(state.abilities.dexterity);
             const current_dex = calculateModifier(payload.abilities.dexterity);
 
-            if (previous_dex <= 0) {
-                state.defence += current_dex;
-            } else {
-                state.defence = state.defence - previous_dex + current_dex;
-            }
+            let defence_bonus_curr = calculateDefenceBonus(current_dex);
+            let defence_bonus_prev = calculateDefenceBonus(previous_dex);
```

Comment:
To stick with the conventions laid out in other parts of the project, might I suggest using `const` and renaming the variables to `current_defence_bonus`  and `previous_defence_bonus` respectively?

Reply:
I think technically we're supposed to be using camel case (which I've neglected sometimes because it's not my normal style...).